packages/edit-site/src/components/navigation-sidebar/navigation-toggle/style.scss

```diff
@@ -62,5 +62,8 @@
 
 .edit-site-navigation-toggle__site-icon {
 	width: $button-size;
+	height: $button-size;
 	border-radius: $radius-block-ui;
+	background-size: cover;
+	margin-top: -1px;
```

Comment:
Same as above, could use a comment and a variable.

Reply:
Switched to using the variable, and added a comment.